lighthouse-core/scripts/gcp-collection/run.sh

```diff
@@ -12,8 +12,9 @@ DIRNAME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 LH_ROOT="$DIRNAME/../../.."
 cd $DIRNAME
 
+GCLOUD_USER=`gcloud config get-value account | awk -F '@' '{print $1}'`
```

Comment:
tbh i'd be content with `whoami` but using the authed acct seems cool too

at the least lets do this
```suggestion
GCLOUD_USER=$(gcloud config get-value account | awk -F '@' '{print $1}')
```

because shellcheck sez so